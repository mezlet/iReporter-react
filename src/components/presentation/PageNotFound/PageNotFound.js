import React from 'react';
import { Link } from 'react-router-dom';
import { Responsive } from 'semantic-ui-react';

function Footer() {
  return (
    <Responsive>
      <div className="pageWrapper">
        <p>PAGE NOT FOUND</p>
        <h4>
          <Link to="/">Redirect to home page</Link>
        </h4>
      </div>
    </Responsive>
  );
}

export default Footer;
