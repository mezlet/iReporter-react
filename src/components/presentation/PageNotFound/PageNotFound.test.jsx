import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from './PageNotFound';

const wrapper = shallow(<PageNotFound />);

describe('<Footer/> rendering', () => {
  it('should render without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
