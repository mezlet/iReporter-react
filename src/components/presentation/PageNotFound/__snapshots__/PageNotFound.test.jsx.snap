// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> rendering should render without crashing 1`] = `ShallowWrapper {}`;
